lib/transport/resource.ex

```diff
@@ -92,6 +92,7 @@ defmodule Transport.Resource do
   def issue_types, do: @issue_types
 
   def valid?(%__MODULE__{} = r), do: r.metadata != nil
+  def valid?(%{} = r), do: Map.has_key?(r, :metadata)
```

Comment:
You need to check `r.metadata` != nil too

Reply:
done. Franchement, ma règle d’or tient plus que jamais : un langage qui du null/nil/none/undef ne mérite que la poubelle.

Reply:
Tu proposes quoi en remplacement du JSON ?

Reply:
On parle de langage, pas de format de données :p